cmd/kube-apiserver/app/server.go

```diff
@@ -82,13 +82,7 @@ cluster's shared state through which all other components interact.`,
 func Run(s *options.APIServer) error {
 	genericvalidation.VerifyEtcdServersList(s.ServerRunOptions)
 	genericapiserver.DefaultAndValidateRunOptions(s.ServerRunOptions)
-	genericConfig := genericapiserver.NewConfig(). // create the new config
-							ApplyOptions(s.ServerRunOptions). // apply the options selected
-							Complete()                        // set default values based on the known values
-
-	if err := genericConfig.MaybeGenerateServingCerts(); err != nil {
```

Comment:
This needed to be done early to ensure the admission client built before `New` would function.  As I recall, you got panics otherwise.